### PR TITLE
added toggle airpods script

### PR DESCRIPTION
**Pull request by itsnwa**
## Description

Connects/disconnects a bluetooth device, in this case AirPods

## Type of change

- [x] New script command

## Dependencies / Requirements

Add your devices MAC address on line 20, you can find this address by opt + clicking the bluetooth menu:
CleanShot 2021-01-27 at 15 29 55@2x

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/system/toggle-airpods.template.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+// @raycast.description Toggle AirPods bluetooth device
+// @raycast.author Nichlas W. Andersen
+// @raycast.authorURL https://github.com/itsnwa
+
+import IOBluetooth
+
+func toggleAirPods() {
+    guard let bluetoothDevice = IOBluetoothDevice(addressString: "Your AirPods MAC address") else {
+        print("Device not found")
+        exit(-2)
```
Comment (unnamedd) [inline commands/system/toggle-airpods.template.swift:22]: @itsnwa here you can use `exit(1)` (fail) and `exit(0)` (success), this is a pattern in the CLI world! ;)
```suggestion
        exit(1)
```

File: commands/system/toggle-airpods.template.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    if !bluetoothDevice.isPaired() {
+        print("Device not paired")
+        exit(-4)
```
Comment (unnamedd) [inline commands/system/toggle-airpods.template.swift:27]: The same here
```suggestion
        exit(1)
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Finally your contribution is here @itsnwa! 🎉  
Nothing major, but two comments! :D

File: commands/system/toggle-airpods.template.swift [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+//
+// Documentation:
+// @raycast.description Toggle AirPods bluetooth device
+// @raycast.author Nichlas W. Andersen
+// @raycast.authorURL https://github.com/itsnwa
+
+import IOBluetooth
+
+func toggleAirPods() {
+    guard let bluetoothDevice = IOBluetoothDevice(addressString: "Your AirPods MAC address") else {
```
Comment (thomaspaulmann) [inline commands/system/toggle-airpods.template.swift:20]: Let's extract the address to a constant and add a comment how to get it.

```suggestion

// Get your device's MAC address by option (⌥) + clicking the bluetooth icon in the menu bar
let deviceAddress = ""

func toggleAirPods() {
    guard let bluetoothDevice = IOBluetoothDevice(addressString: deviceAddress) else {
```

**Review by thomaspaulmann — APPROVED** [review]
Lovely one! Added a comment to get the device address for folks to get started with the command.

**Comment by itsnwa** [comment]
Sweet, thanks @thomaspaulmann @unnamedd 👍

**Pull request merged**